### PR TITLE
Create ADR reconfirming the configuration of fcm ios payload

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Related issue
https://github.com/linagora/tmail-backend/commit/43c49615a06edb850fdd7d4b35fbd05873dfc6af#r141977107

**No push notification case**
```ini
[com.linagora.ios.teammail] Received remote notification request <ID> [ waking: 0, hasAlertContent: 0, hasSound: 0 hasBadge: 0 hasContentAvailable: 1 hasMutableContent: 0 pushType: Background]
```

**Push notification case**
```ini
[com.linagora.ios.teammail] Received remote notification request <ID> [ waking: 0, hasAlertContent: 1, hasSound: 0 hasBadge: 0 hasContentAvailable: 0 hasMutableContent: 1 pushType: Alert]
```

### CODE REVIEW

**Comment by dab246** [comment]
Please attach demo console log for cases if possible

**Review by quantranhong1999 — COMMENTED** [review]
Read it

File: docs/adr/0047-fcm-ios-push-configuration-confirmation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Decision
+When new emails come, the BE will fire notification payload without `content-available` flag.
+
+## Consequences
+1. The iOS will translate the notification message as expected, results in no missing push notification
+2. TODO: Test this changes by:
+   1. Build the iOS app on MacOS with release mode on real iPhone, login and when the app is in email view, put the app in background mode or kill the app. Leave the phone plugged into MacOS machine.
+   2. Open XCode > Window > Devices & Simulators, choose the iPhone currently plugged in, choose "Open Console" and start streaming log
+   3. Use TMail from the web to send an email to self, expected to see `[com.linagora.ios.teammail] Received remote notification request <ID> [ waking: 0, hasAlertContent: 1, hasSound: 0 hasBadge: 0 hasContentAvailable: 0 hasMutableContent: 1 pushType: Alert]` in the Console app after no longer than 1 minute
```
Comment (dab246) [inline docs/adr/0047-fcm-ios-push-configuration-confirmation.md:31]: TMail -> TwakeMail

**Comment by hoangdat** [comment]
read it. Waiting for console log

**Comment by tddang-linagora** [comment]
Log added

File: docs/adr/0047-fcm-ios-push-configuration-confirmation.md (hunk 1 above)
Comment (tddang-linagora) [reply docs/adr/0047-fcm-ios-push-configuration-confirmation.md]: Done

**Comment by quantranhong1999** [comment]
> No push notification case
Push notification case

So the fix works, doesn't it?

**Comment by tddang-linagora** [comment]
> > No push notification case
> > Push notification case
> 
> So the fix works, doesn't it?

Confirmed. Thank you.

**Review by dab246 — APPROVED** [review]
lgtm

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**